lib/sequencescape-api/connection_factory/actions.rb

```diff
@@ -151,7 +151,7 @@ def parse_json_from(response)
   private :parse_json_from
 
   def headers
-    { 'Accept' => 'application/json', 'Cookie' => "WTSISignOn=#{cookie}" }.tap do |standard|
+    { 'Accept' => 'application/json', 'Cookie' => "api_key=#{user_api_key}" }.tap do |standard|
```

Comment:
Apologies, never looked at this code before. Is this is the bit that defines the headers that are passed in requests to Sequencescape?
Will Sequencescape know what to do with a cookie called `api_key`?

Reply:
Yep, Sequencescape has had a fallback to API key for several years now.